format-specs/compatible-parquet.md

```diff
@@ -12,7 +12,7 @@ The core idea of the compatibility guidelines is to have the output match the de
 
 * The geometry column should be named either `"geometry"` or `"geography"`.
 
-* The geometry column should be a `BYTE_ARRAY` with Well Known Binary (WKB) used to define the geometries, as defined in the [encoding](./geoparquet.md#encoding) section of the GeoParquet spec.
+* The geometry column should be a `BYTE_ARRAY` with Well Known Binary (WKB) used to define the geometries, as defined in the [encoding](./geoparquet.md#encoding) section of the GeoParquet spec. Alternatively, the geometry column can be stored according to the Point, MultiPoint, MultiLinestring, or MultiPolygon memory layouts with separated (struct) coordinates as specified in the [GeoArrow format](https://geoarrow.org/format).
```

Comment:
```suggestion
* The geometry column should be a `BYTE_ARRAY` with Well Known Binary (WKB) used to define the geometries, as defined in the [encoding](./geoparquet.md#encoding) section of the GeoParquet spec. Alternatively, the geometry column can be stored according to the Point, MultiPoint, MultiLineString, or MultiPolygon memory layouts with separated (struct) coordinates as specified in the [GeoArrow format](https://geoarrow.org/format).
```
Can it not be interleaved coordinates? Is it because of XYZ/XYM confusion? I don't know how complex we want to make these suggestions, but I'd be content with a suggestion like:

> The interleaved (FixedSizeList) coordinate representation may also be used. In that case, a list size of 2 is expected to represent XY coordinates, a list size of 3 is expected to represent XYZ coordinates, and a list size of 4 is expected to represent XYZM coordinates.